### PR TITLE
Sets

### PR DESCRIPTION
**Pull request by wrackspurt**

### CODE REVIEW

File: mutations.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+Output the sum of elements in set A.
+"""
+
+
+if __name__ == '__main__':
+    try:
+        m = int(input('please, enter the number of elements in set A: '))
+        a = set(int(i) for i in input('please, enter the elements of set A: ').split())
+        n = int(input('please, enter the number of other sets: '))
+        if len(a) == m:
```
Comment (mishindmitry) [inline mutations.py]: if this condition is false - what should happen?

File: mutations.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                elif operation == 'difference_update':
+                    difference_update_set = set(map(int, input('please, enter the elements of the set: ').split()))
+                    a.difference_update(difference_update_set)
+                    if len(difference_update_set) != int(length[0]):
+                        print('something went wrong')
+                elif operation == 'symmetric_difference_update':
+                    symm_diff_set = set(map(int, input('please, enter the elements of the set: ').split()))
+                    a.symmetric_difference_update(symm_diff_set)
+                    if len(symm_diff_set) != int(length[0]):
+                        print('something went wrong')
```
Comment (mishindmitry) [inline mutations.py]: There's a lot of duplicated logic, it seems the only difference is set method called depending on operation. can you please refactor this code?

**Review by mishindmitry — CHANGES_REQUESTED** [review]

File: mutations.py (hunk 1 above)
Comment (wrackspurt) [reply mutations.py]: oh, I'm sorry, I forgot to add 'else'

File: mutations.py (hunk 2 above)
Comment (wrackspurt) [reply mutations.py]: I've already refactored it

**Review by mishindmitry — APPROVED** [review]

**Pull request merged**